addons/aws/1.0.1/install.sh

```diff
@@ -0,0 +1,28 @@
+
+function aws() {
+
+    local dst="$DIR/kustomize/aws"
+    cp "$DIR/addons/aws/1.0.1/kustomization.yaml" "$DIR/kustomize/aws/kustomization.yaml"
+    cp "$DIR/addons/aws/1.0.1/storageclass.yaml" "$DIR/kustomize/aws/storageclass.yaml"
+
+    if [ "$AWS_EXCLUDE_DEFAULT_STORAGE_CLASS" != "1" ]; then
+        insert_resources "$dst/kustomization.yaml" storageclass.yaml
+    fi
+
+    kubectl apply -k "$DIR/kustomize/aws/"
+}
+
+function aws_pre_init() {
+    set_node_name
```

Comment:
its possible that this will interfere with existing installations where the name would change. it would be best to add logic here to account for that. if you dont can you please add a comment explaining the limitations

Reply:
since `hostname -f` is same as `hostname` , added a check to check if both are same , if not, prompt. please see if this makes sense